### ISSUE TITLE
Guidance on breadcrumbs

### ISSUE THREAD

**Issue by joewiz** [uswds/uswds#785]
Does the framework currently provide guidance on breadcrumbs? 

I have been looking for guidance like http://getbootstrap.com/components/#breadcrumbs but couldn't find anything here.

**Comment by joewiz** [comment]
I'm also interested in pagination - both numbered and prev/next.

**Comment by carodew** [comment]
@joewiz These are things that we're considering adding to the design standards in a future release, but we don't have them included at the moment.

**Comment by joewiz** [comment]
Okay, I'll follow developments.

**Comment by Drakenhart** [comment]
Our project has been testing pagination options from both a visual design as well as a 508 compliance side. @carodew would it be helpful to share the data we've discovered here?

**Comment by carodew** [comment]
@Drakenhart Sorry for the delayed reply. Yes, we'd love to hear more about your data. Any information you can provide about the context you used pagination in (what kind of page, etc) (even screenshots) and who you tested with would be extra helpful. Thanks!

**Comment by Drakenhart** [comment]
@carodew I understand. The Holidays complicate online communication. I will see what I can manage.

For context:
We have need for multiple-page table-based lists. Having a single massive page that scrolls more then a screen height only works for screen readers, it's too much information for the user base (all at once).

So as to not hijack this issue further from breadcrumbs, should I open a new one?

**Comment by Drakenhart** [comment]
Side note- on the breadcrumbs topic.

The common design pattern is to use breadcrumbs as a secondary form of supportive navigation. Also the best practices, as I understand them, is that one does not use H-tags in a breadcrumb.

I was curious about the choice, as it is very unique. Where did the inspiration for it come from and why do it this way?

**Comment by carodew** [comment]
@Drakenhart It's probably best to discuss the multi-page tables as a separate discussion. Of course, the length of the scrollable table really depends on the use case. ;)

**Comment by Drakenhart** [comment]
@carodew understood and agreed. I'll create a new item. :)

**Comment by jbruce-gsa** [comment]
@Drakenhart - This is somewhat conjecture, but I think the use of breadcrumbs is a leftover UX design pattern from when people were more accustomed to navigating a file tree on their computer than using a browser with a back button. Further, a lot of older sites had very deep pages buried under multiple levels of files and were easy to get lost in. Information Architecture has moved a long way and made them less desirable or necessary, in my opinion. For the program I am on we use them for helping navigate actual hierarchical data or if the user is more than 3 levels deep (which already goes against the levels provided by the Side Navigation component).

But I'm curious to see where this discussion goes.

**Comment by Drakenhart** [comment]
@jbruce-gsa That's how I understand it as well, and one of the reasons I've found it a good choice for navigational usability. The research done by Nielsen-Norman-Group back in 2007 seems to still hold true, depending on how deep a site goes.

Those who understand their use, use them and to great effect and those that don't ignore them and move along (if there is a main navigation to follow). 

But it also depends on how it's used:
Location- Physical IA-based
Path - user choice action-based (like you might see on an e-commerce site)
Attribute - Category based on user choices (again like you might see on an e-commerce site)

I think my perk of curiosity comes from 2 items specifically. The use of H tags in the breadcrumb trail (not a typical design pattern and not normally suggested as one) is one of them. As well as the large scale size of it so it dominates the page as if it were a main navigation, rather then a more common secondary and supplemental navigation.

**Comment by swirtSJW** [comment]
If we embrace breadcrumbs then is would be good to be aligned with these standards
https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/breadcrumb/index.html

**Comment by joshbruce** [comment]
@swirtSJW: That's an interesting reference, thank you!

I also think, from an information architecture and UX perspective, there might be some room to consider the two as different things.

The trail of breadcrumbs is something that should lead me back to where I started from the way I came - a copy of the history stack...this is literal breadcrumbs.

The docs say "breadcrumbs" is more like a copy of the information architecture...not really breadcrumbs...leaving a user with the back buttom and dropdown in their browser.

Might be my semantic and liguistic brain training to communicate what is really being generated. In the case of the docs, it's more like a linear file tree (mirrors the route/information architecture). Thanks again! Didn't know that existed.
